docker/docker-run.sh

```
@@ -0,0 +1,64 @@
+#!/bin/sh
+#plagiarized from wangxian/alpine-mysql
+if [ -d /app/mysql ]; then
+  echo "[i] MySQL directory already present, skipping creation"
+else
+  echo "[i] MySQL data directory not found, creating initial DBs"
+
+  mysql_install_db --user=root > /dev/null
+
+  if [ "$MYSQL_ROOT_PASSWORD" = "" ]; then
```

I would prefer to be "secure by default".  If the password is missing, instead of defaulting to something insecure, an error should be raised preventing the application from starting.

---

the idea here is that because the db isn't even accessible unless you are able to and do exec into the docker container, there is no way for you to connect to it, it does not matter.